pika/adapters/select_connection.py

```diff
@@ -75,16 +75,6 @@ def _adapter_disconnect(self):
             self.ioloop.remove_handler(self.socket.fileno())
         super(SelectConnection, self)._adapter_disconnect()
 
-    def _flush_outbound(self):
-        """Call the state manager who will figure out that we need to write
-            then call the poller's poll function to force it to process events.
-        """
-        self._manage_event_state()
-        # Force our poller to come up for air, but in write only mode
-        # write only mode prevents messages from coming in and kicking off
-        # events through the consumer
-        self.ioloop.poll(write_only=True)
-
 
 class IOLoop(object):
```

Comment:
Yeah, I did toy with that idea but decided to keep it as is in the end. Still working on trying to get all the ioloops behaving exactly the same and I think it makes sense to maintain an IOLoop class in each at that point. Not had so much time lately though.